discord/appinfo.py

```diff
@@ -68,6 +90,20 @@ def __init__(self, state, data):
         self.bot_require_code_grant = data['bot_require_code_grant']
         self.owner = User(state=self._state, data=data['owner'])
 
+        team = data.get('team')
+        self.team = Team(state, team) if team else None
+
+        self.summary = data['summary']
+        self.verify_key = data['verify_key']
+
+        guild_id = data.get('guild_id')
```

Comment:
There is a utility for this that casts to an integer and what not.
```suggestion
        self.guild_id = utils._get_as_snowflake(data, 'guild_id')
```